src/components/CommandBar/CommandBar.tsx

```diff
@@ -149,6 +151,7 @@ export class CommandBar extends React.Component<ICommandBarProps, ICommandBarSta
                          data-command-key={ index }
                          aria-haspopup={ !!(item.items && item.items.length) }
                          role='menuitem'
+                         aria-label={ `${ item.name }. ${ index === 0 ? this.props.itemNavigationScreenReaderText : '' }` }
```

Comment:
This is for navigation instuction on how to navigate through commands. It only reads when hit the first command. This is in the bug: "1. When tabbing from search to the New menu, we should tell users how to navigate the menu bar by saying something like "Use left and right arrow keys to navigate". We got this feedback directly in a usability study. "